go/storage/api/metrics.go

```diff
@@ -157,6 +115,14 @@ func (w *metricsWrapper) SyncIterate(ctx context.Context, request *IterateReques
 	return res, err
 }
 
+func (w *metricsWrapper) Apply(ctx context.Context, request *ApplyRequest) error {
```

Comment:
Do we no longer want metrics on this, even for local performance reasons?

Reply:
Ooops, yeah not sure how this happened, added it back.